src/OrchardCore/OrchardCore.DisplayManagement.Liquid/OrchardCoreBuilderExtensions.cs

```diff
@@ -79,6 +80,25 @@ public static OrchardCoreBuilder AddLiquidViews(this OrchardCoreBuilder builder)
                         };
                     });
 
+                    o.Scope.SetValue("Environment", new ObjectValue(new LiquidEnvironmentAccessor()));
```

Comment:
Also add some docs about this [to the Liquid Readme](https://github.com/OrchardCMS/OrchardCore/blob/main/src/docs/reference/modules/Liquid/README.md).